src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -160,16 +160,24 @@ static InitChainEntry sInitChain[] = {
 // used in the burning death actionfunc
 static Vec3f D_80B6506C = { 0.0f, 3.0f, 0.0f };
 
-// bytes per limb, used in draw func? bit over my head
-static u8 D_80B65078[BIG_POE_LIMB_MAX] = {
-    -1, 4, -1, 0, -1, 1, -1, 2, 5, 3,
+static s8 sLimbToBodyParts[BIG_POE_LIMB_MAX] = {
+    BODYPART_NONE,      // BIG_POE_LIMB_NONE
+    BIG_POE_BODYPART_4, // BIG_POE_LIMB_FACE
+    BODYPART_NONE,      // BIG_POE_LIMB_LEFT_UPPER_ARM
+    BIG_POE_BODYPART_0, // BIG_POE_LIMB_LEFT_FOREARM
+    BODYPART_NONE,      // BIG_POE_LIMB_RIGHT_UPPER_ARM
+    BIG_POE_BODYPART_1, // BIG_POE_LIMB_RIGHT_FOREARM
+    BODYPART_NONE,      // BIG_POE_LIMB_RIGHT_HAND
+    BIG_POE_BODYPART_2, // BIG_POE_LIMB_LANTERN
+    BIG_POE_BODYPART_5, // BIG_POE_LIMB_HAT_AND_CLOAK
+    BIG_POE_BODYPART_3, // BIG_POE_LIMB_LOWER_ROBE
 };
 
 // used in limbdraw
-static Vec3f D_80B65084[] = {
-    { 2000.0f, 4000.0f, 0.0f },
-    { -1000.0f, 1500.0f, -2000.0f },
-    { -1000.0f, 1500.0f, 2000.0f },
+static Vec3f D_80B65084[BIG_POE_BODYPART_MAX - BIG_POE_BODYPART_6] = {
+    { 2000.0f, 4000.0f, 0.0f },      // BIG_POE_BODYPART_6
+    { -1000.0f, 1500.0f, -2000.0f }, // BIG_POE_BODYPART_7
+    { -1000.0f, 1500.0f, 2000.0f },  // BIG_POE_BODYPART_8
 };
```

Comment:
I also wanted to comment on these types of data, the ones where the bodyPart isn't associated with a limb, but instead is associated with a hard-coded constant. Usually these are at the end, so I was trying to go for `BIG_POE_BODYPART_MAX - BIG_POE_BODYPART_6` as the array size, but maybe `BIG_POE_BODYPART_8 - BIG_POE_BODYPART_6 + 1` makes more sense? Or maybe just leave it as blank

Reply:
I think it would be better to just leave it blank